backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -1094,6 +1095,47 @@ def reader():
             )
             raise
 
+    def delete_file(self, session_id: str, path: str):
+        normalized_path = os.path.normpath(path)
+        if normalized_path == "/workspace":
+            log.warning("Cannot delete the workspace directory")
+            raise ValueError("Cannot delete the workspace directory")
```

Comment:
Please use our exception format: https://dsd-dbs.github.io/capella-collab-manager/development/backend/exception/

Otherwise it's not returned by the API and leads to 500 Internal Server Error.